<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolChords_gui</class>
 <widget class="QWidget" name="ToolChords_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Instrument name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineName">
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sample creation:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboFreq">
       <item>
        <property name="text">
         <string>for every key</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>every 3 keys</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>every 6 keys</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkLoop">
       <property name="text">
        <string>Auto loop</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkStereo">
       <property name="text">
        <string>Stereo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Chord</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="3">
       <widget class="QLabel" name="labelType9">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Seventh</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Ninth</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Third</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="sliderType5">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-1440</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Root key</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Fifth</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="sliderType7">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-1440</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboType3">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>no</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>major</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>minor</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboType1">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>no</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yes</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="sliderType1">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-1440</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboType9">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>no</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>major</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>minor</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="sliderType3">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-1440</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Inversion number</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboType7">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>no</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>minor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>major</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>diminished</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSlider" name="sliderType9">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="minimum">
         <number>-1440</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboType5">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>no</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>perfect</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>diminished</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>augmented</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelType1">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelType5">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelType3">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QSpinBox" name="spinInversions">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="labelType7">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sliderOctave">
          <property name="minimum">
           <number>-3</number>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="2">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">-2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">-3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">-1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">+1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">+2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QLabel" name="label_16">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">+3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Octave</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineName</tabstop>
  <tabstop>checkStereo</tabstop>
  <tabstop>comboFreq</tabstop>
  <tabstop>checkLoop</tabstop>
  <tabstop>comboType1</tabstop>
  <tabstop>sliderType1</tabstop>
  <tabstop>comboType3</tabstop>
  <tabstop>sliderType3</tabstop>
  <tabstop>comboType5</tabstop>
  <tabstop>sliderType5</tabstop>
  <tabstop>comboType7</tabstop>
  <tabstop>sliderType7</tabstop>
  <tabstop>comboType9</tabstop>
  <tabstop>sliderType9</tabstop>
  <tabstop>spinInversions</tabstop>
  <tabstop>sliderOctave</tabstop>
  <tabstop>pushOk</tabstop>
  <tabstop>pushCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
